Don't pass through raw translation values if none match.

Please provide:
What steps should I take to duplicate this problem?
What is the incorrect behavior you are seeing?
What is the correct behavior you want to see?

@jacobthill Here is the proposed solution:

In the config for `cho_language`, you set a default of `NOT FOUND`, like:
```
to_field 'cho_language', extract_xml("#{record}/dc:language", NS), first_only,
         strip, translation_map('marc_languages'),
         default('NOT FOUND')
```

When you run with DebugWriter, you see:
```
http://digital.sabanciuniv.edu/abidindino/mektup/3040200000988.pdf cho_language              NOT FOUND
```

When you run with DlmeJsonResourceWriter, you get:
```

```
[ERROR] Transform produced invalid data.

The errors are: {"cho_language"=>{0=>["must not be one of: NOT FOUND"]}}

The data looked like this:
{
  "cho_title": [
    "Kandilli, 13 Haziran 1979 tarihli Abidin Dino'ya mektup."
  ],
  "cho_language": [
    "NOT FOUND"
  ],
  "cho_description": [
    "Bodrum - Fethi Naci - Ferit Edgü - Vakko - Vitali Hakko - Leyla Umar - \"Cumhuriyet\" - Rasih Nuri İleri. Mektup."
  ],
  "cho_dc_rights": [
    "© Sabancı Üniversitesi Sakıp Sabancı Müzesi. Tüm hakları saklıdır."
  ],
  "cho_format": [
    "pdf"
  ],
  "id": "http://digital.sabanciuniv.edu/abidindino/mektup/3040200000988.pdf",
  "agg_data_provider": "Sakip Sabanci Museum",
  "agg_provider": "Sakip Sabanci Museum"
}
```

How does this work? Is `cho_language` the only field that this would apply to?

--- a/README.md
+++ b/README.md
@@ -118,3 +118,14 @@ first one wins.
 
 ## API Documentation
 https://www.rubydoc.info/github/sul-dlss/dlme-transform
+
+## Transformation notes
+### Unmapped languages
+For the `cho_language` and `cho_edm_type` fields, setting a default of `NOT FOUND` will cause validation to fail when an 
+unmapped language is encountered. For example:
+
+```
+to_field 'cho_language', extract_xml("#{record}/dc:language", NS), first_only,
+         strip, translation_map('marc_languages'),
+         default('NOT FOUND')
+```

--- a/lib/dlme_json_schema.rb
+++ b/lib/dlme_json_schema.rb
@@ -12,14 +12,14 @@ DlmeJsonSchema = Dry::Validation.Schema do
   optional('cho_date') { array? { each(:str?) } }
   optional('cho_dc_rights') { array? { each(:str?) } }
   optional('cho_description') { array? { each(:str?) } }
-  optional('cho_edm_type') { array? { each(:str?) } }
+  optional('cho_edm_type') { array? { each(:str?, excluded_from?: ['NOT FOUND']) } }
   optional('cho_extent') { array? { each(:str?) } }
   optional('cho_format') { array? { each(:str?) } }
   optional('cho_has_part') { array? { each(:str?) } }
   optional('cho_has_type') { array? { each(:str?) } }
   optional('cho_identifier') { array? { each(:str?) } }
   optional('cho_is_part_of') { array? { each(:str?) } }
-  optional('cho_language') { array? { each(:str?) } }
+  optional('cho_language') { array? { each(:str?, excluded_from?: ['NOT FOUND']) } }
   optional('cho_medium') { array? { each(:str?) } }
   optional('cho_provenance') { array? { each(:str?) } }
   optional('cho_publisher') { array? { each(:str?) } }

